composer.json

```diff
@@ -18,22 +18,11 @@
         "ext-simplexml": "*",
         "guzzle/guzzle": "^2.8|^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.0|^3.0",
+        "symfony/config": "^2.6|^3.0",
```

Comment:
not cool. The project itself works on 2.0, why raise that? CircleCI shortcomings aren't the reason.